daemon/src/maker_inc_connections.rs

```diff
@@ -102,43 +111,40 @@ impl Actor {
         let transport_state = Arc::new(Mutex::new(transport_state));
 
         let (read, write) = stream.into_split();
-        let read = FramedRead::new(read, wire::EncryptedJsonCodec::new(transport_state.clone()))
-            .map_ok(move |msg| FromTaker { taker_id, msg })
-            .map(forward_only_ok::Message);
+        let mut read =
+            FramedRead::new(read, wire::EncryptedJsonCodec::new(transport_state.clone()))
+                .map_ok(move |msg| FromTaker { taker_id, msg });
```

Comment:
I think now that we are not using the `Stream` with `attach_stream`, it would be more idiomatic to construct the `FromTaker` message just upon `taker_msg_channel.send()`.